webcompat/webhooks/helpers.py

```diff
@@ -9,7 +9,11 @@
 import re
 import requests
 
+from db import issue_db
```

Comment:
Gotcha. I'm going to close this PR so someone doesn't accidentally merge this.

Feel free to send another one once it's ready to be merged (and feel free to ping for feedback on commit comments or in #165). :heart: